packages/admin/cms-admin/src/generator/future/generator.ts

```diff
@@ -45,7 +46,7 @@ export type GridColumnConfig<T> = (
     | { type: "dateTime" }
     | { type: "staticSelect"; values?: string[] }
     | { type: "block"; block: ImportReference }
-) & { name: keyof T } & DataGridSettings;
+) & { name: UsableFields<T> } & DataGridSettings;
```

Comment:
Why "usable"?

Reply:
because the other fields of the entity given (`[ForGrid]UsableFields[of]<T>`) can't be used. e.g. 
```
product {
    name // is usable
    manufacturer { // is not usable
        name // is usable
        ...
    }
    ...
}
```